src/index.html

```diff
@@ -14,8 +14,45 @@
     >
     <title>Product cards</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet"> 
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img 
+      class="card__image" 
+      src="./images/imac.jpeg" 
+      alt="Apple iMac" 
+      >	
```

Comment:
oh no
attributes indentation
```suggestion
      <img 
        class="card__image" 
        src="./images/imac.jpeg" 
        alt="Apple iMac" 
      >	
```